.github/workflows/build.yml

```diff
@@ -129,16 +127,13 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: "8.3"
+          php-version: "8.2"
           ini-values: "phar.readonly=0,zend.exception_ignore_args=Off"
           coverage: none
           tools: box
 
       - name: Build the PHAR
-        run: |
-          composer config platform.php 7.2.34
```

Comment:
Resolving dependencies on PHP 8.2 without a platform config ensuring compat with PHP 8.1 is a no-go, as it would not guarantee that the phar is compatible with PHP 8.1 (it would likely install Symfony 7 which requires PHP 8.2)

Reply:
That's my bad, did the initial review on my phone and forgot box needs 8.2 and we're supporting 8.1 still. Will fix.